wire-ios-sync-engine/Source/Synchronization/EventProcessor.swift

```diff
@@ -86,8 +86,10 @@ actor EventProcessor: UpdateEventProcessor {
     }
 
     private func enqueueTask(_ block: @escaping @Sendable () async throws -> Void) async throws {
+        defer { processingTask = nil }
+
         processingTask = Task { [processingTask] in
-            _ = await processingTask?.result
+            _ = try await processingTask?.value
```

Comment:
I'm not sure if I understand this change. The `try` here inside the task is not handled, is it?

Reply:
It's not but the error is captured as the result of the task if we throw inside it.

We await the result below.